esx_circleprogressbar/fxmanifest.lua

```diff
@@ -0,0 +1,17 @@
+fx_version 'cerulean'
+game 'gta5'
+
+author 'Its_Cehic'
+version '1.0'
+description 'Circle Progress Bar - Test Version'
+-- Created and Edited by Its_Cehic
+
+client_script 'script.js'
+server_script 'progress.lua'
+
+ui_page 'index.html'
```

Comment:
these need to be nui/filename not just filename due to file paths. and in the files {} beneath

Reply:
I tested! Here is error about that is not defined the "document".
Here is the photo:
![image](https://github.com/esx-framework/esx_core/assets/128829862/87e7f004-285f-43ea-8646-6f2bbcee1705)